docs/asb/README.md

```diff
@@ -0,0 +1,134 @@
+
+![Service Provider scenarios](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/comit-network/xmr-btc-swap/04a65005428c34891451762a94035df5754e6c19/docs/asb/diagrams/cli-asb-overview.puml)
+
+Eventually, more elaborate discovery mechanims can be added.
+
+### Setup Details
+
+In order to understand the different components of the ASB and CLI better here is a component diagram showcasing the ASB and CLI setup using public Bitcoin and Monero infrastructure:
+
+![Service Provider scenarios](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/comit-network/xmr-btc-swap/7afa953efd398d65d54b4488ec11d40a8647ab24/docs/asb/diagrams/cli-asb-components-asb-pub-nodes.puml)
+
+Contrary, here is a diagram that showcases an ASB running it's own blockchain infrastructure:
+
+![Service Provider scenarios](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/comit-network/xmr-btc-swap/7afa953efd398d65d54b4488ec11d40a8647ab24/docs/asb/diagrams/cli-asb-components-asb-self-hosted.puml)
+
+The diagram shows that the `asb` group (representing the `asb` binary) consists of three components:
+
+1. Monero Wallet
+2. Bitcoin Wallet
+3. ASB
+
+The `ASB` depicted in the diagram actually consists of multiple components (protocol impl, network communication, ...) that sums up the functionality to execute concurrent swaps in the role of Alice.
+
+#### Monero Wallet Setup
+
+The ASB uses the running Monero wallet RPC to create / open Monero wallets.
+Currently you cannot connect to an existing Monero wallet, but the ASB will create the wallet `asb-wallet` upon intial startup.
+In order to accept trades with a CLI you will have to send XMR to that wallet.
+The wallet's address is printed upon startup of the ASB.
+Currently the `asb-wallet` does not have a password.
+
+Upon startup of the ASB the `asb-wallet` is opened in the wallet RPC.
+You can then interact with the wallet RPC for basic wallet management as well.
+
+#### Bitcoin Wallet Setup
+
+The ASB has an internall managed Bitcoin wallet.
+The Bitcoin wallet in created upon initial startup and stored in the data folder of the ASB (configured through initial startup wizard).
+
+Interactions with the Bitcoin wallet are still work in progress:
+
+- https://github.com/comit-network/xmr-btc-swap/issues/368
+
+#### Price Discovery
+
+In order to be able to trade, the ASB must define a price to be able to agree on the amounts to be swapped with a CLI.
+Currently we use a spot-rpice mode, i.e. the ASB dictates the price to the CLI.
+
+A CLI can connect to the ASB at any time and request a quote for buying XMR, for a given BTC amount.
+The ASB then returns the current price and the maximum amount tradeable.
+
+The maximum amount tradeable can be configured with the `--max-buy-btc` parameter.
+The parameter is currently not included in the config, but has to be set upon startup of the application.
+It defaults to `0.005` BTC.
```

Comment:
I think we can remove that.

The first sentence is obvious from the fact that it is a commandline argument. The second one can be seen from `--help` and including it here is just prone to be outdated.